mysite/missions/templates/missions/shell/ls.html

```diff
@@ -54,7 +54,7 @@
     <p>There are many other options available. For more options, type
     <kbd class="kbd">ls --help</kbd> in your shell and you'll get a list of
     available options along with a short description for each option.
-    (if you are on zsh type <kbd class="kbd">man ls</kbd>)</p>
+    (if you are on zsh type or OS X bash <kbd class="kbd">man ls</kbd>)</p>
```

Comment:
maybe `if you are on zsh or OS X bash, type <kbd class="kbd">man ls</kbd>)</p>`?

(Otherwise it sounds like `zsh type` might be yet another kind of shell!)